Resources/Textures/Objects/Weapons/Guns/Battery/laser_gun.rsi/meta.json

```diff
@@ -1,7 +1,7 @@
 {
     "version": 1,
     "license": "CC-BY-SA-3.0",
-    "copyright": "Taken from cev eris at https://github.com/discordia-space/CEV-Eris/blob/master/icons/obj/guns/energy/retro.dmi",
+    "copyright": "Taken from ds13 at https://github.com/DS-13-Dev-Team/DS13/blob/dev/icons/obj/gun.dmi",
```

Comment:
Should have a specific commit (yeah the old one also had this issue).

Reply:
found the initial commit on the Polaris server